utils/draw_text_on_image.py

```diff
@@ -0,0 +1,9 @@
+import cv2
+import numpy as np
+
+def draw_text_on_image(images, text):
+    output_images = np.zeros(shape=images.shape)
```

Comment:
I wonder if it would be better to add some white below the image and write on that to make it easier to read. (use the same shape as input image, double the height). Set to 1 the lower part.